.env.example

```diff
@@ -2,6 +2,7 @@ API_URL=
 API_KEY=
 GTFS_URL=
 BUSLOC_URL=
+SENTRY_DSN=
```

Comment:
This file is meant for local setup. Assuming we only want prod to send to sentry, this should go in the README instead.